src/subdomains/core/buy-crypto/process/services/buy-crypto-preparation.service.ts

```diff
@@ -67,21 +67,12 @@ export class BuyCryptoPreparationService {
       try {
         if (entity.cryptoInput && (!entity.cryptoInput.isConfirmed || !entity.cryptoInput.amlCheck)) continue;
 
-        const inputCurrency = entity.cryptoInput?.asset ?? (await this.fiatService.getFiatByName(entity.inputAsset));
         const inputReferenceCurrency =
           entity.cryptoInput?.asset ?? (await this.fiatService.getFiatByName(entity.inputReferenceAsset));
 
         const inputReferenceAssetChfPrice = await this.pricingService.getPrice(inputReferenceCurrency, fiatChf, false);
 
-        const { minVolume } = await this.transactionHelper.getTxFeeInfos(
-          entity.inputReferenceAmount,
-          inputCurrency,
-          inputReferenceCurrency,
-          entity.target.asset,
-          entity.paymentMethodIn,
-          CryptoPaymentMethod.CRYPTO,
-          entity.user,
-        );
+        const minVolume = await this.transactionHelper.getMinVolumeIn(inputReferenceCurrency, false);
 
```

Comment:
Do we need to use the inputCurrency to get the min volume (it might depend on the currency)?